src/status_im2/contexts/onboarding/enter_seed_phrase/style.cljs

```diff
@@ -9,6 +11,24 @@
    :right            0
    :background-color colors/neutral-80-opa-80-blur})
 
+(def form-container
+  {:flex               1
+   :padding-horizontal 20
+   :padding-vertical   12})
+
+(def header-container
+  {:flex-direction  :row
+   :justify-content :space-between})
+
+(def word-count-container
+  {:justify-content :flex-end
+   :margin-bottom   2})
+
 (def input-container
   {:height            120
+   :margin-top        12
    :margin-horizontal -20})
+
+(def continue-button {:margin-top :auto})
```

Comment:
Consider adding `margin-bottom` using `(safe-area/get-bottom)` to handle the safe area.

Reply:
yeah could you elaborate on what is auto?

Reply:
@ajayesivan I think bottom is not required in this case :thinking: have you noticed it's required for this screen? thanks

@flexsurfer 
`:auto` in `maring-top` makes the continue button go to the bottom of the screen.
About `:auto` in margins, it creates as much space as possible.

Reply:
I agree! will fix it! thanks!

Reply:
@ajayesivan Solved!